front/src/Phaser/Game/GameMap.ts

```diff
@@ -155,6 +185,54 @@ export class GameMap {
         }
     }
 
+    /**
+     * We user Tiled Objects with type "zone" as zones with defined x, y, width and height for easier event triggering.
+     */
+    private triggerZonesChange(): void {
+        const zones = this.tiledObjects.filter(object => object.type === "zone");
+
+        // P.H. NOTE: We could also get all of the zones and add properties of occupied tiles to them, so we could later on check collision by using tileKeys
```

Comment:
No need for that for now (I understand this could be slightly faster, but maybe it is a bit premature)